### DISCUSSION TITLE
What does it mean? help me plz!

### DISCUSSION THREAD

**Discussion by lorenzomonta** [Homebrew/discussions#376] [Casks] [answered]
Sorry but I can't understand what happened with the Python package, could someone help me? thanks

![page 1](https://user-images.githubusercontent.com/48455647/103181243-7a130980-489f-11eb-83dc-6a674cfbe11b.png)
![page 2](https://user-images.githubusercontent.com/48455647/103181245-84350800-489f-11eb-81d9-c140e350791d.png)

**Comment by gromgit** [comment, marked as answer]
Homebrew tried to install the `python@3.9` formula, but found that you'd already linked several _system_ Python programs (`2to3`, `python3`, etc.) into `/usr/local/bin`.

Since they are all symlinks, it should be safe to do what `brew` says:
```
To force the link and overwrite all conflicting files:
  brew link --overwrite python@3.9
```

Sidenote: In future, when asking for command-line help, please _copy-and-paste_ screen output (which we can in turn copy, test, and modify) rather than taking screenshots (which we can only look at and retype manually).

**Reply by lorenzomonta** [reply]
Thank you for your reply.
Maybe it is not necessary because at the end of the second screen, after the ==> Summary, the installation of python 3.9.1_2 version is successful becouse It has removed /python@3.9/3.9.1_1 directory.
It is correct?

Thank you

**Reply by gromgit** [reply]
Read the `brew link` error message again. Like I said, some of the symlinks that `brew install` complained about are to the **macOS** Python installation. In particular, when `pip3` points to Homebrew but `python3` points to macOS, that will probably just lead to "why can't my scripts find the modules I just installed?!?!"

**Reply by lorenzomonta** [reply]
You are right! thank you
